app/component/ItineraryList/ItineraryList.js

```diff
@@ -262,11 +213,42 @@ function ItineraryList(
         </div>
       );
     }
+    if (itineraries.length === 0) {
+      return (
+        <div className="summary-no-route-found" style={{ marginTop: 0 }}>
+          <div
+            className={cx('flex-horizontal', 'summary-notification', 'info')}
+          >
+            <Icon
+              className={cx('no-route-icon', 'info')}
+              img="icon-icon_info"
+              color="#0074be"
+            />
+            <div>
+              <div className="in-the-past">
+                <FormattedMessage
+                  id="router-only-walk-title"
+                  defaultMessage=""
+                />
+              </div>
+              <FormattedMessage
+                id="router-only-walk"
+                defaultMessage={
+                  'Unfortunately no routes were found for your journey. ' +
+                  'Please change your origin or destination address.'
+                }
+              />
+            </div>
+          </div>{' '}
```

Comment:
What is this empty space for?

Reply:
Apparently for nothing. Found and fixed plenty of those.